main.tf

```diff
@@ -183,6 +280,7 @@ resource "aws_ecs_service" "service" {
   deployment_minimum_healthy_percent = var.deployment_minimum_healthy_percent
   deployment_maximum_percent         = var.deployment_maximum_percent
   health_check_grace_period_seconds  = var.lb_arn == "" ? null : var.health_check_grace_period_seconds
+  platform_version                   = (var.create_efs_vol == true ? "1.4.0" : "LATEST")
```

Comment:
This may force a new resource if create efs volume is false. Perhaps a better approach could be:

`var.create_efs_vol == true ? "1.4.0" : null`